datafusion/sqllogictest/test_files/array.slt

```diff
@@ -5169,8 +5169,9 @@ false false false true
 true false true false
 true false false true
 false true false false
-false false false false
-false false false false
```

Comment:
The test result does not look correct, because it ignore some null rows in between